packages/amplify-graphiql-explorer/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "noImplicitAny": false,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "useUnknownInCatchVariables":false,
```

Comment:
shouldn't this be pulled in from the base config?

Reply:
same as above